vite.config.ts

```diff
@@ -1,8 +1,24 @@
-import { defineConfig } from 'vite';
+import path from 'path';
+
 import react from '@vitejs/plugin-react';
 import svgr from 'vite-plugin-svgr';
 
+// Normally there should be a `defineConfig` around this
+// Configuration object, however this is not possible
+// because we need to add a "test" object that is
+// not supported on the regular `defineConfig`
+// and the `defineConfig` from viTest does not
+// work for some reason.
 // https://vitejs.dev/config/
-export default defineConfig({
```

Comment:
We had to remove `defineConfig` here because it does not support the `test` block. ViTest has a `defineConfig` function but that has it's own set of issues that prevent use.